operators/src/source/gdal_source/mod.rs

```diff
@@ -450,22 +547,32 @@ where
             start.elapsed()
         );
 
-        // TODO: evaluate if there are GeoTransforms with positive y-axis
-        // The "Pixel-space" starts at the top-left corner of a `GeoTransform`.
+        // A `GeoTransform` maps pixel space to world space.
+        // We are not aware of spatial reference systems where the x-axis points tot he right.
```

Comment:
typo "tot he"

Reply:
i think i removed that in one of the commits